config/etc/system-image/writable-paths

```diff
@@ -82,9 +67,8 @@
 # dbus bus policy
 /etc/dbus-1/system.d                    auto                    persistent  transition  none
 /etc/modprobe.d                         auto                    synced      none        none
+# FIXME: do we still need ppp on the image?
```

Comment:
We probably need /etc/ppp writable still, as mm writes there /etc/ppp/resolv.conf